pkg/kubelet/cm/devicemanager/plugin/v1beta1/server.go

```diff
@@ -134,6 +143,7 @@ func (s *server) Stop() error {
 	s.grpc.Stop()
 	s.wg.Wait()
 	s.grpc = nil
+	s.setHealthy(false)
```

Comment:
this will not be ideal. It will be a vary bad experience if during tearing the kubelet down, systemd will decide that it is not healthy and will kill it. Let's not mark it unhealthy on termination. You can put a comment here that explains that we DO NOT set unhealthy on termination.

```suggestion
        // during kubelet termination we do not need the registration server and we consider kubelet as healthy even when it is down.
	s.setHealthy(true)
```

Reply:
get